package.json

```diff
@@ -16,7 +16,10 @@
     "build-tests": "mkdir -p dist/ && browserify test/karma-qunit-shim.js test/*-test.js -u test/karma.conf.js -s videojs-dock -d -o dist/tests.js",
     "watch-tests": "mkdir -p dist/ && watchify test/karma-qunit-shim.js test/*-test.js -u test/karma.conf.js -s videojs-dock -v -d -o dist/tests.js",
     "test": "jshint src/**.js test/**.js index.js && npm run build-tests && karma start test/karma.conf.js --single-run",
-    "prepublish": "npm run build",
+    "predist": "npm run babel",
```

Comment:
We can also use babelify and point browserify at `src/index.js`. Doing it this way means we can just `browserify .` and have it use the `main` field of this file.